ProffieOS.ino

```diff
@@ -1328,6 +1332,18 @@ public:
   static const char* response_footer() { return "-+=END_OUTPUT=+-\n"; }
 };
 
+#ifdef USE_TEENSY4
+class Serial4Adapter {
+public:
+  static void begin() { Serial4.begin(115200); }
+  static bool Connected() { return true; }
+  static bool AlwaysConnected() { return true; }
+  static Stream& stream() { return Serial3; }
```

Comment:
That would give more flexibility in pin-layout. Although, looking at the suggested pin definition for a Teensy4.0, Serial1 and Serial4 are the only candidates. It would mean re-arranging of the amplifier enable, and bladePowerPin1.
I2C, SPI and I2C are more or lest fixed to certain pins, and also have to make sure your Neopixel aren interfeing wiht any of the 'fixed' pins.

Is making the #define for SERIAL_CONSOLE_PORT  something that you are willing to write and integrate into ProffieOS? (for now, I think the define for Serial4adapter option will work as well)

Reply:
SERIAL_CONSOLE_PORT is trivial to add, I can do that part easily.